components/multiselect/option/option.jsx

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {displayEntireNameForUser} from 'utils/utils.jsx';
+
+import ProfilePicture from 'components/profile_picture.jsx';
+
+export default class Option extends PureComponent {
```

Comment:
I very much like refactoring the options out into their own component, but like the other stuff, this should be specific to users instead of being a part of the regular MultiSelectList